packages/matter.js/src/behavior/Behavior.ts

```diff
@@ -95,6 +95,14 @@ export abstract class Behavior {
         return this.#agent.context;
     }
 
+    get exchange() {
```

Comment:
If use is the exchange is limited could we leave this out and just do error checking when we read from the context?

Reply:
Alright. Will adjust